ramlfications/parser.py

```diff
@@ -927,24 +928,50 @@ def resp_headers(headers):
         def resp_body(body):
             """Set response body."""
             body_list = []
-            if body.get("schema"):
-                schema = body.get("schema")
-                example = body.get("example")
-            elif body.get("application/json"):
-                schema = body.get("application/json", {}).get("schema")
-                example = body.get("application/json", {}).get("example")
-            else:
-                schema = {}
-                example = {}
-            b = Body(
-                mime_type="application/json",
-                raw=body,
-                schema=load_schema(schema),
-                example=load_schema(example),
-                form_params=None,
-                config=root.config
-            )
-            body_list.append(b)
+            default_body = {}
+            for (key, spec) in body.items():
+                if key not in MEDIA_TYPES:
+                    # if a root mediaType was defined, the response body
+                    # may omit the mime_type definition
+                    if key == 'schema':
```

Comment:
I see you may have noticed the RAML spec's inconsistencies too :) 

It [says](http://raml.org/spec.html#body) that the `body` param should be a key/value map, where the key is a valid media type.  But in some cases - especially within the examples they use elsewhere in the spec, as well as some [examples](https://github.com/mulesoft/api-console/tree/master/dist/examples) they publish, media types are completely omitted.  

I think handling this the way you did is the pragmatic approach, and hopefully when they release 1.0 of the spec, they'll clear it up a bit.